addons/binding/org.openhab.binding.roku/src/main/java/org/openhab/binding/roku/internal/protocol/RokuCommands.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.roku.internal.protocol;
+
+import static org.openhab.binding.roku.RokuBindingConstants.*;
+
+import java.io.IOException;
+
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link RokuCommands} is responsible sending a command to a Roku device via {@link RokueNetworkCalls}.
+ *
+ * @author Jarod Peters - Initial contribution
+ */
+public class RokuCommands {
+
+    private final Logger logger = LoggerFactory.getLogger(RokuCommands.class);
+
+    private RokuNetworkCalls rnc;
```

Comment:
Consider making this final, since you assign it via the constructor, so always just once

Reply:
I opted to delete this class instead.